charts/consul/values.yaml

```diff
@@ -66,7 +66,7 @@ global:
   # image: "hashicorp/consul-enterprise:1.10.0-ent"
   # ```
   # @default: hashicorp/consul:<latest version>
-  image: docker.mirror.hashicorp.services/hashicorppreview/consul:1.17-dev
+  image: docker.mirror.hashicorp.services/hashicorppreview/consul:1.16.1
```

Comment:
I guess this should still be `1.17-dev`

Reply:
yes, it was for local testing, reverting